src/directory/managed_directory.rs

```diff
@@ -28,6 +30,13 @@ fn is_managed(path: &Path) -> bool {
         .unwrap_or(true)
 }
 
+/// Returns true iff the file require a footer.
+fn require_footer(path: &Path) -> bool {
+    is_managed(path) &
```

Comment:
`is_managed` is very misleading. It seems to imply it is part of the files marked as managed, as in "part of the files listed in `.managed.json`.